triage/index.html

```diff
@@ -26,6 +26,7 @@ <h1 class="clearoptions">Kubernetes Aggregated Failures</h1>
 </span>
 <br><br>
 <form id="options" onchange="rerender();">
+<input id="date" type="hidden">
```

Comment:
@rmmh If I understand correctly this PR adds an input field to enter a date to filter the results on. Any reason why this text field is kept hidden?